swagger.go

```diff
@@ -96,7 +100,9 @@ func CustomWrapHandler(config *Config, h *webdav.Handler) gin.HandlerFunc {
 			c.Writer.Write([]byte(doc))
 			return
 		default:
+			locker.Lock()
 			h.ServeHTTP(c.Writer, c.Request)
+			locker.Unlock()
```

Comment:
```suggestion
			locker.RLock()
			h.ServeHTTP(c.Writer, c.Request)
			locker.RUnlock()
```
Please replace it with reading lock/unlock in order to leverage concurrent HTTP requests.